File: Ejercicio3.py
"""3.-Crear un programa en el cual, Ingrese 10 valores,y proceda a calcular su suma y promedio."""

sum=0
for i in range(10):
    num = int(input(f"Ingrese el numero {i+1} :  "))
    sum = sum + num

prom = sum/10

print("La suma de los numeros es : ", sum)
print("El promedio de los numeros es : ", prom)

print("PROGRAMA FINALZIADO!!")

File: Ejercicio3.js
/* 3.-Crear un programa en el cual, Ingrese 10 valores, 
y proceda a calcular su suma y promedio.
*/

var sum=0

for( var i=0;i<10;i++){
    var num=Number(prompt("Ingrese el numero "+ (i+1)+ ": "))
    sum=sum+num;

}

var prom=sum/10

window.alert(" La suma de los numeros es: "+ sum +"\n El promedio de los numero es: "+prom)
